src/frontend/web_application/src/scenes/DevicesSettings/presenter.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import { Trans } from '@lingui/react';
+import { Trans } from '@lingui/macro'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
very important!

when a `Trans` includes a component there is 2 possibilities: 
* declare all the things
* use `macro` a dev dependencies instead that will transpil into a full declaration including the importation itself:

```
// src
import { Trans } from '@lingui/macro';
//...
<Trans id="foo"><p>Hello<br />{name}</p></Trans>
```

```
// semi-transpiled
import { Trans } from '@lingui/react';

//...
<Trans id="foo" components={[<p/>, <br />]} values={{ name: 'foo' }} defaults="<0>Hello<1 />{name}</0>" />
```

without macro we must use semi-transpiled format

cf. https://lingui.js.org/ref/macro.html#jsx-macros